examples/spring-boot-minimal-kotlin/build.gradle

```diff
@@ -0,0 +1,47 @@
+buildscript {
```

Comment:
Since this is a Kotlin example, how about making the build script Kotlin too

Reply:
Good idea :)

Reply:
Fixed